components/ui/tabcounter/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,5 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
     <color name="mozac_ui_tabcounter_default_tint">#FF272727</color>
+    <color name="mozac_ui_tabcounter_default_text">#20123A</color>
```

Comment:
Not sure if we should be passing colors and styles into the Tab Counter Menu

Reply:
I think we can make this customizable later, if we need it. For now only Fenix uses it.